### PR TITLE
3.6.0/jsonschema

### PR DESCRIPTION
**Pull request by TedZen**
merge JsonSchemaDataClassGenerator.kt into DataClassGeneratorByJSONSchema.kt with kotlinpoet-removing

### CODE REVIEW

**Comment by TedZen** [comment]
@rodion-m hi there, I just merged your JsonSchemaDataClassGenerator.kt into DataClassGeneratorByJSONSchema.kt , can you review this PR If it's convenient.

File: src/main/kotlin/wu/seal/jsontokotlin/model/classscodestruct/EnumClass.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import wu.seal.jsontokotlin.utils.toAnnotationComments
+import java.lang.StringBuilder
+
+/**
+ * Created by ted on 2020/3/14 18:14.
+ */
+data class EnumClass(
+        override val name: String,
+        val xEnumNames: List<String>?,
+        override val generic: KotlinClass,
```
Comment (wuseal) [inline src/main/kotlin/wu/seal/jsontokotlin/model/classscodestruct/EnumClass.kt]: NOGeneric Class should not have this property for usage all right?

**Comment by rodion-m** [comment]
> @rodion-m hi there, I just merged your JsonSchemaDataClassGenerator.kt into DataClassGeneratorByJSONSchema.kt , can you review this PR If it's convenient.

Hello! I don't know how to check the correctness of this PR. I only can suggest to test all json-schema cases, collected by me in file `JsonSchemaGeneratorTest.kt` and if it's all ok, then OK!
Also I have a question for @wuseal why you did decide to refuse Kotlin poet? Did you consider an idea to use it in whole project?

**Comment by wuseal** [comment]
@rodion-m Hi, happy hear your new voice. all cases in `JsonSchemaGeneratorTest.kt` have been checked that can pass.
>  why you did decide to refuse Kotlin poet?

For two reason:
* I want to define kotlin class data struct ourselves.
* KotlinPoet library's size is too big within the kotlin reflection library, and we use standalone kotlin library to keep plugin stably.

**Comment by rodion-m** [comment]
> @rodion-m Hi, happy hear your new voice. all cases in `JsonSchemaGeneratorTest.kt` have been checked that can pass.
> 
> > why you did decide to refuse Kotlin poet?
> 
> For two reason:
> 
> * I want to define kotlin class data struct ourselves.
> * KotlinPoet library's size is too big within the kotlin reflection library, and we use standalone kotlin library to keep plugin stably.

Ok, thank you for answer. I'll will be happy to try new generator in the plugin :)
Thanks a lot to you and @TedZen for what you have implemented it!

**Pull request merged**